src/plugins/displayLayout/components/TelemetryView.vue

```diff
@@ -0,0 +1,166 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
+ <template>
+    <div class="c-telemetry-view"
+         :style="styleObject">
+        <div v-if="showLabel"
+              class="c-telemetry-view__label">
+            <div class="c-telemetry-view__label-text">{{ item.domainObject.name }}</div>
+        </div>
+
+        <div v-if="showValue"
+              class="c-telemetry-view__value"
+              :class="[telemetryClass]">
+            <div class="c-telemetry-view__value-text">{{ telemetryValue }}</div>
+        </div>
+    </div>    
+ </template>
+
+<style lang="scss">
+    @import '~styles/sass-base';
+
+    .c-telemetry-view {
+        display: flex;
+        align-items: stretch;
+
+        > * {
+            // Label and value holders
+            flex: 1 1 auto;
+            display: flex;
+            flex-direction: row;
+           // justify-content: center;
+            align-items: center;
+            overflow: hidden;
+            padding: $interiorMargin;
+
+            > * {
+                // Text elements
+                @include ellipsize();
+            }
+        }
+
+        > * + * {
+            margin-left: $interiorMargin;
+        }
+
+        .c-frame & {
+            @include abs();
+            border: 1px solid transparent;
+        }
+    }
+</style>
+
+ <script>
+    export default {
+        inject: ['openmct'],
+        props: {
+            item: Object
+        },
+        computed: {
+            showLabel: function () {
+                let displayMode = this.item.config.alphanumeric.displayMode;
+                return (displayMode === 'all' || displayMode === 'label') ? true : false;
```

Comment:
don't need the ternary operator when the return value is already a boolean.

Reply:
Good point. Will fix it.